gig-marketplace.md

```diff
@@ -0,0 +1,57 @@
+* It is difficult for managers to approximate what the time/effort commitment will be for employees taking on InnerSource project tasks.
+* The less insight a manager has into the effort and accomplishment of their employees in InnerSource projects, the more unlikely it is that he can reward their employees for their InnerSource work.
+* The transparency and reputational aspect of the code submissions within a large company should work to keep the code quality high for InnerSource contributions.
+* Individual employees may commit themselves to InnerSource project tasks without first getting approval from their manager or having the skills or time required to complete it.
+
+## Context
+
+* Part of a manager's job is tracking, estimating and understanding use of employee time
+* Managers have given tacit acceptance that their employees should take part in InnerSource projects
+* There is no corporate policy on employee’s involvement in InnerSource projects
+* Employees desire the freedom to work on projects that are of interest to or would professionally benefit them
+* Employees desire being recognized for their efforts when contributing to InnerSource projects
+* Employees desire exposure to different areas within and to work with a larger pool of people across the company
+
+## Solutions
+
+Create a “gig” based intranet site where individuals can advertise their skills and areas of interest and project owners can advertise opportunities for collaboration.  Each gig should describe exactly the skill and time requirements needed so those can be easily and clearly communicated to an employee’s direct management.  A points based system should be created to reward and track an employee’s involvement in a gig. Points accrued by completing a gig can then be used as performance management criteria and to garner insight in to the areas of expertise that exist within an organization. All gigs are initially vetted with the Gig owner to determine that the employee has the prerequisite skills and allotted time and needed access to complete the gig.  Gigs can also factor in soft skills such as garnering help organizing a group event or requesting help editing/writing a report or requests for mentorship etc.
+
+## Resulting Context
+
+The InnerSource Gig Marketplace has vastly increased the number of possibilities for InnerSource opportunities. The self-directed nature of Gig Marketplace has enhanced job satisfaction amongst employees by allowing them a level of choice in the work they perform and with whom they can partner across the company. Managers are able to better estimate and track their employee’s time commitments with respect to InnerSource projects, recognize their individual efforts and use the completion of gigs as a way of validating their specific skillsets.  Managers are also able to leverage any existing downtime their employees may be experiencing by allowing them to pivot to work available in the Gig Marketplace.
+
```

Comment:
Seems like there should be a linkage/mention of the InnerSource Portal and an explanation of how the Gig Marketplace and InnerSource portal relate to each other.

Reply:
OK. The two patterns could be used independently though. Would it be enough to mention that the two patterns can be used to better effect in combination?